pkg/virt-config/config-map.go

```diff
@@ -59,6 +59,8 @@ const (
 	MigrationCompletionTimeoutPerGiB         int64  = 800
 	DefaultMachineType                              = "q35"
 	DefaultCPURequest                               = "100m"
+	DefaultMemoryRequest                            = "8Mi"
```

Comment:
I have now seen multiple times the request that `DefaultMemoryRequest` should be set to `1G`, also users have reported crashing VMIs, and it turned out that they did not set memory. I think even you had a reasonable value in the PR initially. I think at this point I would be ok with a reasonable default. Either that or we set no default. I can't even remember  why we started  with this "strange" default value.

Reply:
Yeah, you're right - I also changed the default memory request to 1G as part of this PR but I dropped this change later. The thing that bothered me, and please corrects me if I'm wrong, is that this value is used for scheduling. So setting it to 1G may hinder memory over-commit, right?

Therefore I dropped this change because I wanted to propose to add a memory property to the VMI , like the additional CPU properties that the VMI has, and treat this memory request as "guaranteed memory" in oVirt. We could then have some (configurable?) logic that sets the guaranteed memory according to the general memory field and vice versa (in oVirt the memory defaults to 4 times the guaranteed memory IIRC). What do you think?

Reply:
I would prefer to handle that separately though - as it makes sense (to me :) ) to make it configurable and this PR doesn't change the existing default.

Reply:
agreed. this is a discussion that we can handle outside of this PR. I like that this PR doesn't functionally change any defaults.

Reply:
Works for me